### PR TITLE
feat/1862 add a PR template

### PR DESCRIPTION
**Pull request by mshidlov**
Adding new PR template to the repository

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!--
+Thank you for contributing to Amplication :)
+
+PLEASE, GO THROUGH THESE STEPS BEFORE YOU SUBMIT A PR!
```
Comment (MoFoGo) [inline .github/PULL_REQUEST_TEMPLATE.md:4]:
```suggestion
PLEASE, GO THROUGH THESE STEPS BEFORE SUBMITTING A PR!
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+Make sure that:
+
+1. There is an open issue for this PR. If not, please open one before submitting your changes. Before proceeding, any change needs to be discussed (You can skip this if you're fixing a typo or adding an app to the Showcase).
+
+2. You have done your changes in a separate branch. Branches MUST have descriptive names that start with either the `fix/[issue #]-` or `feature/[issue #]-` prefixes. Good examples are: `fix/404-signin-issue` or `feature/201-new-templates`.
+
+3. Your pull request MUST NOT target this repository's `master` branch. You probably want to target a branch name that starts with `release/` instead.
```
Comment (MoFoGo) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
3. Your pull request MUST NOT target this repository's `master` branch. It is recommended that you target a branch name that starts with `release/` instead.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+4. You give a descriptive title to your PR.
```
Comment (MoFoGo) [inline .github/PULL_REQUEST_TEMPLATE.md:14]:
```suggestion
4. You gave a descriptive title to your PR.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+5. You provide enough information about your changes so that others can review your pull request.
```
Comment (MoFoGo) [inline .github/PULL_REQUEST_TEMPLATE.md:16]:
```suggestion
5. You have provided enough information about your changes for others to review your pull request.
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (mshidlov) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: maybe `You are giving a descriptive title to your PR.` will be better?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (mshidlov) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: maybe `You are providing enough` will be better here as well?

Comment (MoFoGo) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: Agreed

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (MoFoGo) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: Agreed

**Pull request merged**